src/App.tsx

```diff
@@ -1,167 +1,38 @@
-import { Loader } from './components/Loader';
-
+import React from 'react';
 import './App.scss';
-
-export const App = () => (
-  <div data-cy="app">
-    <nav
-      data-cy="nav"
-      className="navbar is-fixed-top has-shadow"
-      role="navigation"
-      aria-label="main navigation"
-    >
-      <div className="container">
-        <div className="navbar-brand">
-          <a className="navbar-item" href="#/">
-            Home
-          </a>
-
-          <a
-            className="navbar-item has-background-grey-lighter"
-            href="#/people"
-          >
-            People
-          </a>
+import { NavLink, Outlet } from 'react-router-dom';
+import classNames from 'classnames';
+
+export const App = () => {
+  const getClassNames = ({ isActive }: { isActive: boolean }) =>
+    classNames('navbar-item', { 'has-background-grey-lighter': isActive });
```

Comment:
1. Use `Type` for ur function
2. Change syntax for more readability

```suggestion
  const getClassNames = ({ isActive }: YourType) => (
    classNames('navbar-item', {
      'has-background-grey-lighter': isActive 
    })
  );
```